<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRUDAccidentes</class>
 <widget class="QMainWindow" name="CRUDAccidentes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>id Accidente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha Accidentes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lugar Accidente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IdAccidente">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnAgregar">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnBuscar">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnModificar">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnEliminar">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255,125,100);
}</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LugarAcc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="FechaAcc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>9</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="currentSectionIndex">
     <number>0</number>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>9</month>
      <day>1</day>
     </date>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
